int32 container
---
bool found
geometry_msgs/Pose container_pose
